### PR TITLE
Fix update/switch and restart prompt on Unity 2020.2+

### PR DESCRIPTION
**Pull request by jimon**

### CODE REVIEW

File: Packages/com.unity.inputsystem/CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ however, it has to be formatted properly to pass verification tests.
 ### Fixed
 
 - Fixed precompiled layouts such as `FastKeyboard` leading to build time regressions with il2cpp (case 1283676).
+- Fixed update/switch and restart prompt on Unity 2020.2+ ([case 1292513](https://issuetracker.unity3d.com/issues/input-system-after-package-install-the-update-slash-switch-and-restart-prompt-does-not-appear)).
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/CHANGELOG.md]: Suggest: "Fixed restart prompt after package installation not appearing on Unity 2020.2+ (...)."

File: Packages/com.unity.inputsystem/Tests/IntegrationTests/IntegrationTests.cs [hunk 2]
```diff
@@ -83,7 +87,42 @@ public void Integration_CanSendAndReceiveEvents()
         }
     }
 
-    #if UNITY_STANDALONE_WIN || UNITY_EDITOR_WIN
+#if UNITY_EDITOR
+
+    [Test]
+    [Category("Integration")]
+    public void Integration__InEditor_AbleToChangePlayerSettings()
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/Tests/IntegrationTests/IntegrationTests.cs]: Minor: extra underscore `__`. Naming-wise, maybe `Integration_CanChangeInputBackendPlayerSettingInEditor`.

**Review by Rene-Damm — APPROVED** [review]

File: Packages/com.unity.inputsystem/Tests/IntegrationTests/IntegrationTests.cs (hunk 2 above)
Comment (jimon) [reply Packages/com.unity.inputsystem/Tests/IntegrationTests/IntegrationTests.cs]: fixed

File: Packages/com.unity.inputsystem/CHANGELOG.md (hunk 1 above)
Comment (jimon) [reply Packages/com.unity.inputsystem/CHANGELOG.md]: fixed

**Pull request merged**